### PR TITLE
token classification models artifacts update

### PR DESCRIPTION
**Pull request by ekmb**

### CODE REVIEW

**Review by ericharper — APPROVED** [review]
Thanks!

File: Jenkinsfile [hunk 1]
```diff
@@ -886,7 +886,10 @@ pipeline {
             trainer.gpus=[0,1] \
             trainer.accelerator=ddp \
             +trainer.fast_dev_run=true \
-            exp_manager=null'
+            +exp_manager.explicit_log_dir=/home/TestData/nlp/token_classification_punctuation/output
+            python examples/nlp/token_classification/punctuation_capitalization_evaluate.py \
+            pretrained_model=/home/TestData/nlp/token_classification_punctuation/output/checkpoints/Punctuation_and_Capitalization.nemo && \
+            rm -rf /home/TestData/nlp/token_classification_punctuation/output/*'
```
Comment (okuchaiev) [inline Jenkinsfile:892]: looks like CI does not like star?

File: Jenkinsfile [hunk 2]
```diff
@@ -886,7 +886,10 @@ pipeline {
             trainer.gpus=[0,1] \
             trainer.accelerator=ddp \
             +trainer.fast_dev_run=true \
-            exp_manager=null'
+            +exp_manager.explicit_log_dir=/home/TestData/nlp/token_classification_punctuation/output
```
Comment (ericharper) [inline Jenkinsfile:889]: missing a `\` at the end of this line.

**Comment by ekmb** [comment]
@okuchaiev @ericharper we need to enable the full CI before merging this

**Comment by ericharper** [comment]
> @okuchaiev @ericharper we need to enable the full CI before merging this

Looks like the Jenkinfile reverted to main. I replaced with v1.0.0 again.

**Review by ericharper — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**